### ISSUE TITLE
Printing of results

### ISSUE THREAD

**Issue by annelorejp** [nnzhan/Graph-WaveNet#33]
Hi,

Currently I'm working on the Graph-WaveNet code with my own dataset (predicting the amount of patients occupying a bed in a hospital). 

There is one thing I don't fully understand from the code.

In this line of code (test.py line 100-104): 
<img width="612" alt="afbeelding" src="https://user-images.githubusercontent.com/44172501/164236037-133d3c9b-e3f2-4a09-8633-9f752090ffcd.png">

    y12 = realy[:,99,11].cpu().detach().numpy()
    yhat12 = scaler.inverse_transform(yhat[:,99,11]).cpu().detach().numpy()

    y3 = realy[:,99,2].cpu().detach().numpy()
    yhat3 = scaler.inverse_transform(yhat[:,99,2]).cpu().detach().numpy()

Can anyone explain to me what they are doing in this piece of code? And what the difference is between y12 and y3?

I really hope someone can help me! :)

Thanks in advance!

**Comment by zowb** [comment]
It is the prediction at time step 3 and 12 respectively.

**Comment by annelorejp** [comment]
> 

Thanks for your reply!
So there is predicted based on the time steps before AND the training model which uses the training data?
OR
Are the predictions only based on the couple of time steps before?

**Comment by packer-c** [comment]
Same question. 
What does 99 mean?

**Comment by packer-c** [comment]
oh, I knew that 99 represents 99th node.
